FAQ.md

```diff
@@ -31,3 +31,18 @@ install.packages("tinytex")
 tinytex::install_tinytex()
 ```
 
+## Windows user and have a lot of problems getting packages to install ?
+
+> ⚠️ Note that **Windows is not officially supported in the course** but we will try our best to help you get your setup (in Windows) working. This is because of two main reasons:
+> 1. `Stan`, the probabilistic programming language which we will later on use in the course, doesn't have good support for Windows. 
```

Comment:
It should be said that "`Stan`, the probabilistic programming language requires C++ compiler toolchain which is not available by default in Windows (blame Microsoft)."

Reply:
Just updated the FAQ. The new version should cover most of the points we discussed here.